src/api/resolvers/CCIMSSchema.ts

```diff
@@ -0,0 +1,44 @@
+import { GraphQLSchema } from "graphql"
+import mutation from "./mutation";
+import query from "./query";
+import typeImports from "./typeImports";
+
+let ccimsSchema = new GraphQLSchema({
+    types: typeImports,
+    mutation,
+    query
+});
+export default ccimsSchema;
+
+/*

```

Comment:
why is there a huge amount of commented out code?

Reply:
70b2ef5